src/ardrive.ts

```diff
@@ -126,13 +139,26 @@ export class ArDrive extends ArDriveAnonymous {
 		});
 	}
 
+	async encryptedFileSize(filePath: string, drivePassword: string, driveId: string, fileId: string): Promise<number> {
```

Comment:
This does seem rather intense to do this here for a price and then throw away the buffer, especially in a bulk upload sense.

I wonder what the results of this function are when graphed out. Is there a standard amount of bytes added on, or does it depend on the file size?

And why do we accept a driveId and fileId here instead of just defining the fake IDs here, do different characters of the same uuid length produce different encrypted sizes?

Reply:
All valid. Am going to look into AES GCM block sizes.

Reply:
There's hope: https://stackoverflow.com/questions/3283787/size-of-data-after-aes-cbc-and-aes-ecb-encryption

Note that we need to sort out the padding sizes.